tests/UnitTests/depthmcu_ut/depthmcu_ut.cpp

```diff
@@ -495,7 +495,7 @@ TEST_F(depthmcu_ut, depthmcu_get_serialnum_non_ascii)
 
     // Create a serial number with non-ACII characters
     char mockSerialNumber[20];
-    memcpy(mockSerialNumber, "12\145\t67890\0WXYZ", sizeof("1234567890\0WXYZ"));
+    memcpy(mockSerialNumber, "12\145\t67890\0WXYZ", sizeof("12\145\t67890\0WXYZ"));
```

Comment:
GCC Warning: `forming offset 16 is out of the bounds [0, 15]`

Reply:
Hats off to GCC for catching this!

Reply:
What is going on with formatting in this string?